modern/src/DevicesList.js

```diff
@@ -26,8 +28,72 @@ const useStyles = makeStyles(() => ({
     height: '25px',
     filter: 'brightness(0) invert(1)',
   },
+  batteryText: {
+    fontSize: '0.75rem',
+    fontWeight: 'normal',
+    lineHeight: '0.875rem'
+  },
+  green: {
+    color: theme.palette.common.green,
+  },
+  red: {
+    color: theme.palette.common.red,
+  },
+  gray: {
+    color: theme.palette.common.gray
+  }
 }));
 
+const OnlineStatus =({ status }) => {
```

Comment:
Instead of this can we just have a function that returns style class?

Reply:
I have tried a simple function, but we can't use the hook in the simple function.

Reply:
You can just return the name of the style you want to use.

Reply:
Done.